providers/go-feature-flag/src/main/java/dev/openfeature/contrib/providers/gofeatureflag/GoFeatureFlagProvider.java

```diff
@@ -476,52 +474,6 @@ private <T> T convertValue(Object value, Class<?> expectedType) {
         return (T) objectToValue(value);
```

Comment:
Good catch, I missed that the SDK was implementing that directly.

Reply:
It was just recently added.